app/controllers/case_information_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CaseInformationController < ApplicationController
+  before_action :authenticate_user
+
+  def new
+    @prisoner = OffenderService.new.get_offender(nomis_offender_id_from_url).data
+  end
+
+  def create
+    CaseInformation.create!(
+      nomis_offender_id: case_information_params[:nomis_offender_id],
+      tier: case_information_params[:tier],
+      case_allocation: case_information_params[:case_allocation]
+    )
+
+    redirect_to allocations_path
```

Comment:
This probably needs to be `allocations_path(anchor:'awaiting-tiering')`

Reply:
👍